addons/binding/org.openhab.binding.miele/README.md

```diff
@@ -14,6 +14,7 @@ Oven
 Microwave/Oven combination
 Tumbledryer
 Washingmachine
+Coffeemaker
```

Comment:
Miele calls these: Coffee machines Maybe follow that name (may also in the rest of the code). Also this list seems to be in alphabetical order, so the Coffee machines should not be the last item. While in text this is show as a list, when viewed as html it just places them after each other making it hard to read. Maybe add some markdown styling to make it show up as a list.

Reply:
@maplesteve can you process this remark?